src/main/scala/rna16s.scala

```diff
@@ -1,49 +1,22 @@
 package era7bio.db.rna16s
 
-import ohnosequences.cosas._, types._, klists._
-import ohnosequences.statika._
-import ohnosequences.fastarious._, fasta._, ncbiHeaders._
 import ohnosequences.blast.api._
-import better.files._
+import ohnosequences.awstools.s3._
 
-trait AnyBlastDB {
+import rnaCentralTable._
 
-  lazy val name: String = getClass.getName
-  // TODO sequence header stuff; this should be lcl()
-  // this is what will be added to the fasta headers
-  lazy val ncbiID = ncbiHeaders.name(name)
 
-  // TODO S3 Object address; AnyData?
-  val sourceFasta: String
-  val dbType: BlastDBType
-}
-
-// TODO this should get the input fasta from somewhere, download it and run makeblastdb etc
-// blast should be a dependency
-trait GenerateBlastDB[DB <: AnyBlastDB] extends Bundle() {
+case object rna16sDB extends AnyBlastDB {
+  val dbType = BlastDBType.nucl
 
-  val db: DB
+  private val ver = "5.0"
+  private val s3folder = S3Folder("resources.ohnosequences.com", s"rnacentral/${ver}")
 
-  // TODO build NCBI header stuff with lcl
-  def processInput(inputFasta: File, drop: FASTA.Value => Boolean, outputFasta: File): File = {
+  private[rna16s] val sourceFasta: S3Object = s3folder / s"rnacentral.${ver}.fasta"
+  private[rna16s] val sourceTable: S3Object = s3folder / s"id2taxa.active.${ver}.tsv"
 
-    fasta
-      .parseFastaDropErrors(inputFasta.lines)
-      .filterNot(drop)
-      .map(fa => fa.update(header := FastaHeader(s"${fa.getV(header).id}|${toHeader(db.ncbiID)}")))
-      .appendTo(outputFasta)
-
-    outputFasta
+  val predicate: Row => Boolean = { row =>
+    row(GeneName) == "16s"
+    // TODO: what else?
```

Comment:
I just pushed. We also need to filter out short sequences, which requires access to the fasta value.